docs/sources/as-code/observability-as-code/schema-v2/annotations-schema.md

```diff
@@ -39,7 +39,8 @@ The configuration for the list of annotations that are associated with the dashb
         "enable": false,
         "hide": false,
         "iconColor": "",
-        "name": ""
+        "name": "",
```

Comment:
Thank you so much for updating this page, but don't worry it. We're getting rid of it in the near future in favour of directing folks to the Swagger docs. It's already wildly out of date.